javascript-data-model/main.js

```diff
@@ -0,0 +1,28 @@
+/* exported orderHistory */
+
+var orderHistory = [
+  {
+    itemNames: 'JavaScript for Impatient Programmers',
```

Comment:
Look at the output for each item. They include the item title, cost, and an image of the item. Is a string description of the item name going to be enough data to do that?